babel.config.js

```diff
@@ -2,39 +2,34 @@ module.exports = api => {
   // NB: This function can be called without an api argument, e.g. by bin/bundle
 
   const presets = []
-  const plugins = [
-    ['@babel/plugin-transform-arrow-functions'],
-    ['@babel/plugin-transform-block-scoping'],
-    ['@babel/plugin-transform-classes', { loose: true }],
-    ['@babel/plugin-transform-computed-properties', { loose: true }],
-    ['@babel/plugin-transform-destructuring', { loose: true }],
-    ['@babel/plugin-transform-member-expression-literals'],
-    ['@babel/plugin-transform-property-literals'],
-    ['@babel/plugin-transform-parameters', { loose: true }],
-    ['@babel/plugin-transform-shorthand-properties'],
-    ['@babel/plugin-transform-spread', { loose: true }],
-    ['@babel/plugin-transform-template-literals', { loose: true }],
-    ['@babel/plugin-proposal-object-rest-spread', { loose: true }],
-    ['@babel/syntax-object-rest-spread']
-  ]
-
-  if (api && !api.env('test')) {
-    api.cache(false)
-  }
+  const plugins = []
 
   if (api && api.env('test')) {
     presets.push(
-      [
-        '@babel/preset-env',
-        {
-          targets: {
-            node: 'current'
-          }
-        }
-      ],
-      '@babel/preset-typescript'
+      '@babel/preset-typescript',
+      'module:metro-react-native-babel-preset'
```

Comment:
This config is a bit weird – this means we now basically rely on this preset (`module:metro-react-native-babel-preset`) to compile all of our non-react native tests too. I couldn't figure out a way to do:

```
if react native tests, load these plugins/presets
else if browser/node tests, load these plugins/presets
else load these plugin presets // <- this is what actually gets used for the production bundle
``